src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsMeta_SummaryAdmin.cshtml

```diff
@@ -1,13 +1,14 @@
 @using OrchardCore.ContentManagement
-
+@using System.Globalization
 @{
     ContentItem contentItem = Model.ContentItem;
+    var modifiedUtc = contentItem.ModifiedUtc?.ToString("yyyy-MM-ddTHH:mm:sszzz", CultureInfo.InvariantCulture);
```

Comment:
Why we didn't use the current culture?

Reply:
> Why we didn't use the current culture?

https://www.geeksforgeeks.org/html-time-datetime-attribute/#:~:text=The%20datetime%20Attribute%20in,%2DDDThh%3Amm%3AssTZD.